packages/dcos-integration-test/extra/test_dcos_checks.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import uuid
+
+
+def test_dcos_checks_detect_ip(dcos_api_session):
+    cmd = [
+        "/opt/mesosphere/bin/dcos-checks",
+        "--role",
+        "agent",
+        "components",
+        "--port",
+        "61001",
+        "--detect-ip",
+        "/etc/hosts",
+        "--exclude=dcos-checks-poststart.service,dcos-checks-poststart.timer"
+    ]
+    test_uuid = uuid.uuid4().hex
+    job = {
+        'id': 'test-dcos-checks-detect-ip-' + test_uuid,
+        'run': {
+            'cpus': .1,
+            'mem': 128,
+            'disk': 0,
+            'cmd': ' '.join(cmd)
+        }
+    }
+
+    with pytest.raises(Exception):
+        dcos_api_session.metronome_one_off(job)
```

Comment:
Can we run a second job that runs the check against `/opt/mesosphere/bin/detect_ip`, and verifies it succeeds? That way we can be more certain that this isn't "passing" because e.g. Metronome is broken.

Reply:
I was thinking branden was requesting an additional step in the same test case. Having it as an additional test case (like you have it is now) is fine.

Reply:
Yeah either approach is fine with me.